src/devtools/client/debugger/src/actions/logExceptions.js

```diff
@@ -21,10 +23,12 @@ export function setupExceptions(store) {
  */
 export function logExceptions(shouldLogExceptions) {
   return (dispatch, getState, { client }) => {
+    dispatch(exceptionLogpointErrorCleared());
+
     return dispatch({
-      type: "LOG_EXCEPTIONS",
-      shouldLogExceptions,
       [PROMISE]: client.logExceptions(shouldLogExceptions),
+      shouldLogExceptions,
+      type: "LOG_EXCEPTIONS",
```

Comment:
:question: Is whatever new auto-sorting setup actually sorting object keys?  If so, that seems heavy-handed

Reply:
It is!